src/OrchardCore.Modules/OrchardCore.Email/Controllers/AdminController.cs

```diff
@@ -49,7 +49,7 @@ public async Task<IActionResult> Index()
             return View();
         }
 
-        [HttpPost]
+        [HttpPost, ActionName(nameof(Index))]
```

Comment:
Won't this break https://github.com/Garpo001/OrchardCore/blob/e353bb0da2c5b5dbe2a217b3ea6d9813a80ac9b5/src/OrchardCore.Modules/OrchardCore.Email/Views/Admin/Index.cshtml#L5 ? @agriffard

Reply:
OK, Action also changed in the form.

Reply:
How can this work if the action is "Index" and the form action is "IndexPost"?

Reply:
Ok you guys renamed it back to index again. But this time fixed the action on the controller. Kind of the same but I don't understand why it kept changing to "index" for the form action.